### PR TITLE
Linked the database to PHP page, created an array with the list from …

### PR DESCRIPTION
**Pull request by CosminDarius1**
…database

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+$db->setAttribute(PDO::ATTR_ERRMODE,PDO::ERRMODE_EXCEPTION);
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE,PDO::FETCH_ASSOC);
+
+$querry = $db->prepare("SELECT `name`,`year`,`director`,`genre` FROM `movies`");
+$querry->execute();
+$data = $querry->fetchAll();
+
+
+
+array_push($data);
```
Comment (Joshsilas) [inline index.php]: This isn't doing anything

**Review by Joshsilas — CHANGES_REQUESTED** [review]

File: index.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+$db = new PDO('mysql:host=db;dbname=collectorapp','root','password');
+$db->setAttribute(PDO::ATTR_ERRMODE,PDO::ERRMODE_EXCEPTION);
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE,PDO::FETCH_ASSOC);
+
+$querry = $db->prepare("SELECT `name`,`year`,`director`,`genre` FROM `movies`");
```
Comment (alexjames4) [inline index.php]: incorrect spelling of query

File: index.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+$querry->execute();
```
Comment (alexjames4) [inline index.php]: see above comment

**Review by alexjames4 — CHANGES_REQUESTED** [review]
a few typos in your code that code be an issue further down the line

File: index.php [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+$db = new PDO('mysql:host=db;dbname=collectorapp','root','password');
+$db->setAttribute(PDO::ATTR_ERRMODE,PDO::ERRMODE_EXCEPTION);
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE,PDO::FETCH_ASSOC);
+
+$query = $db->prepare("SELECT `name`,`year`,`director`,`genre` FROM `movies`");
+$query->execute();
+$data = $querry->fetchAll();
```
Comment (alexjames4) [inline index.php]: spelling error

**Review by alexjames4 — CHANGES_REQUESTED** [review]
one more spelling error noticed :)

File: index.php (hunk 4 above)
Comment (alexjames4) [reply index.php]: i'd also argue $data isnt a good name for a variable

**Review by alexjames4 — APPROVED** [review]
lovely stuff

**Pull request merged**